### DISCUSSION TITLE
Defenses

### DISCUSSION THREAD

**Discussion by PDNinja333** [nroutasuo/level13#208] [Q&A] [answered]
I have a very high chance of raids but have no idea how to get defenses. Please help

**Comment by nroutasuo** [comment, marked as answer]
You'll be able to unlock Fortifications through research. In the meanwhile, avoid building unnecessary buildings since they increase raid danger.

**Comment by ACorridorWanderer** [comment]
Yeah, agreed.

(don't forget to fully explore places.)
